### PR TITLE
Removes data attribute from resource node (content metadata).

### LINKED ISSUE #3084: Normalize content metadata invalid and/or deprecated resource attributes
For example: https://argo.stanford.edu/view/druid:zn580kw6428 uses resource attributes that are no longer valid. Instead, this druid should get a valid resource type.

### PR DESCRIPTION
**Pull request by justinlittman**
closes #3084

## Why was this change made?
Normalization

## How was this change tested?
Unit

### CODE REVIEW

**Comment by justinlittman** [comment]
Before:
```
Status (n=100000; not using Missing for success/different/error stats):
  Success:   94844 (94.922%)
  Different: 5074 (5.078%)
  Mapping error:     0 (0.0%)
  Create error:     0 (0.0%)
  Normalization error:     0 (0.0%)
  Missing from cache:     0 (0.0%)
  Unmapped:     0 (0.0%)
  Expected unmapped:     82 (0.082%)
  Bad cache:     0 (0.0%)
```

After:
```
Status (n=100000; not using Missing for success/different/error stats):
  Success:   94844 (94.922%)
  Different: 5074 (5.078%)
  Mapping error:     0 (0.0%)
  Create error:     0 (0.0%)
  Normalization error:     0 (0.0%)
  Missing from cache:     0 (0.0%)
  Unmapped:     0 (0.0%)
  Expected unmapped:     82 (0.082%)
  Bad cache:     0 (0.0%)
```

File: spec/services/cocina/normalizers/content_metadata_normalizer_spec.rb [hunk 1]
```diff
@@ -268,4 +268,36 @@
...
+            <file preserve="yes" shelve="yes" deliver="yes" size="919405" mimetype="application/pdf" id="v212299_pdf.pdf">
+              <checksum type="md5">174a44a8406b14949de14853612e4eb6</checksum>
+              <checksum type="sha1">258881e0f2293b90779cbc35e7f463882cc2bbf3</checksum>
+            </file>
+          </resource>
+        </contentMetadata>#{'    '}
+      XML
+    end
+
+    it 'removes data and id attributes' do
```
Comment (ndushay) [inline spec/services/cocina/normalizers/content_metadata_normalizer_spec.rb:287]: id attributes?   And what is that after the closing tag of `</contentMetadata>` ?

**Review by ndushay — CHANGES_REQUESTED** [review]
it would be good to provide a druid for the spec, especially since the run against 100,000 objects has no occurrences ...

**Comment by justinlittman** [comment]
druid:zn580kw6428 is an example of this.

File: spec/services/cocina/normalizers/content_metadata_normalizer_spec.rb (hunk 1 above)
Comment (justinlittman) [reply spec/services/cocina/normalizers/content_metadata_normalizer_spec.rb]: Thanks. Fixed.

**Review by ndushay — APPROVED** [review]

**Comment by ndushay** [comment]
merge when green (currently failing)

**Pull request merged**